app/web3/loadIcoParamsFromContract.ts

```diff
@@ -16,4 +16,3 @@ export async function loadIcoParamsFromContract(address: string) {
     endDate: endDate.toISOString(),
   };
 }
-export default loadIcoParamsFromContract;
```

Comment:
Why did we take off the default export?

Reply:
because it was not doing anything? it's already exported a named export. Generally, I wouldn't rely on default exports too much. In react ecosystem only containers use default exports. I guess we could create some guidelines, when to use them and when don't